components/Link/src/stories/bem.stories.mdx_

```diff
@@ -0,0 +1,388 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+
+import { Link } from "../index";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS/Navigation/Link"
```

Comment:
```suggestion
  title="CSS/Navigation/Link"
  chromatic: { viewports: [1768, 1280, 768, 360] },
```

Reply:
Bedankt voor je suggestie! Is dit gerelateerd aan de Storybook 7 upgrade, of kan dit in een losse PR alvast gemerged worden?

Reply:
Nee dit werk al in de huidige storybook.